easyrules-core/src/main/java/org/easyrules/api/RulesEngine.java

```diff
@@ -96,4 +96,5 @@
      */
     void clearRules();
 
+    void unregisterRuleByName(String r1);
```

Comment:
This can be `unregisterRule(String ruleName)` to overload `unregisterRule(Object rule)` method